pkg/v1/tkg/test/tkgctl/shared/e2e_common.go

```diff
@@ -150,6 +155,41 @@ func E2ECommonSpec(context context.Context, inputGetter func() E2ECommonSpecInpu
 		By(fmt.Sprintf("Waiting for workload cluster %q nodes to be up and running", clusterName))
 		framework.WaitForNodes(framework.NewClusterProxy(clusterName, tempFilePath, ""), 2)
 
+		if input.E2EConfig.ManagementClusterOptions.Plan == "devcc" {
```

Comment:
fly-by comment: it is quite conceivable we might test with the `prodcc` plan in the future, so we should also account for it here.